src/components/dashboard/FeedItems/FeedModalItem.js

```diff
@@ -38,7 +38,7 @@ const FeedModalItem = (props: FeedEventProps) => {
       itemType={itemType}
       onClose={buttonPress}
       showJaggedEdge={showJaggedEdge}
-      fullHeight={true}
+      fullHeight
```

Comment:
why did you remove this? i think modal items should take all height and not shrink if less content

Reply:
fullHeight={true}  ====  fullHeight .  in props. it is just short variant

Reply:
@sirpy this change nothing - its the same code, just shortened